_data/cryptocurrencies.yml

```diff
@@ -229,6 +229,13 @@ websites:
       software: Yes
       doc: https://support.uphold.com/hc/en-us/articles/206119683-How-does-phone-number-verification-work-
 
+    - name: WEX (BTC-E)
```

Comment:
I guess it depends on how you look at it ... legally its a new company that bought the assets of the old company. To users, the UI looks exactly the same and their balances generally transferred over. So for me I've listed the old name so that people mentally link them.

Reply:
Alright that make sense.